snuba/query/processors/timeseries_processor.py

```diff
@@ -57,11 +57,23 @@ def __init__(
                                 [
                                     String("toStartOfHour"),
                                     String("toStartOfMinute"),
+                                    String("toStartOfDay"),
```

Comment:
Please add it here as well. 
https://github.com/getsentry/snuba/blob/master/snuba/query/processors/timeseries_processor.py#L208